<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraInformation</class>
 <widget class="QDialog" name="CameraInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>204</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>217</width>
    <height>204</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>217</width>
    <height>204</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Camera Data</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #E3EAFF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="NameHint">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="camearaNameInfo">
       <property name="text">
        <string>nameField</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="viewingAngleHint">
       <property name="text">
        <string>Viewing angle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="viewingAngleInfo">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="positionHint">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="positionInfo">
       <property name="text">
        <string>{0, 0, 0}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="rotationAngleHint">
       <property name="text">
        <string>Rotation angle</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="rotationAngleInfo">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="visibilityRangeHint">
       <property name="text">
        <string>Visibility range</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="visibilityRangeInfo">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="changeSettingsButton">
     <property name="styleSheet">
      <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
     </property>
     <property name="text">
      <string>Change settings </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
